buildout.cfg

```diff
@@ -7,3 +7,6 @@ plone.restapi =
 
 # 3.0 drops python2 compatibility
 pyroma = 2.6.1
+
+# Improvements to the JSONField
+plone.schema = 1.3.0
```

Comment:
@sneridagh is that a hard requirement? If that's the case we have to add it to setup.py. If it works with older versions as well we should keep things as they are because version pins in setup.py are annoying.

Reply:
No, not hard (in normal code) but the tests will fail if it's not present.